sentinelsat/sentinel.py

```diff
@@ -1457,3 +1457,11 @@ def placename_to_wkt(placename):
     footprint = "ENVELOPE({}, {}, {}, {})".format(minX, maxX, maxY, minY)
     placeinfo = feature["properties"]["display_name"]
     return footprint, placeinfo
+
+
+def is_wkt(possible_wkt):
+    SET_OF_ACCEPTABLE_SHAPES = "POLYGON((", "MULTIPOLYGON(((", "ENVELOPE("
```

Comment:
According to the Sci-Hub docs only Polygon and Point are accepted: https://scihub.copernicus.eu/twiki/do/view/SciHubUserGuide/OpenSearchAPI#Discover_the_products_over_a_pre

Reply:
I've made a quick fix here, but I could make it more of a regex if you want.

Reply:
I would not bother with restricting the set of WKT types and would use a pretty general regex pattern instead:
```regex
\s*[A-Za-z]+\s*\(.+\)\s*
```

Reply:
Great input, thanks. just changed it to `\s*[A-Za-z]*\s*\(.+\)\s*` so it can accept the Point type as it says in the scihub docs.